ArchiSteamFarm/Steam/Bot.cs

```diff
@@ -875,78 +803,13 @@ public sealed class Bot : IAsyncDisposable {
 			ArchiLogger.LogChatMessage(true, message, chatGroupID, chatID);
 
 			string? steamMessagePrefix = ASF.GlobalConfig != null ? ASF.GlobalConfig.SteamMessagePrefix : GlobalConfig.DefaultSteamMessagePrefix;
-			ushort maxMessageLength = (ushort) (MaxMessageLength - ReservedMessageLength - (steamMessagePrefix?.Length ?? 0));
 
-			// We must escape our message prior to sending it
-			message = Escape(message);
+			await foreach (string messagePart in SteamChatMessage.GetMessageParts(message, steamMessagePrefix).ConfigureAwait(false)) {
+				if (!await SendMessagePart(() => ArchiHandler.SendMessage(chatGroupID, chatID, messagePart)).ConfigureAwait(false)) {
```

Comment:
Same as for the other call to `SendMessagePart(...)`.

Reply:
No.